### DISCUSSION TITLE
Liquid Converter CLI tool Vs Liquid Convertor Service/API

### DISCUSSION THREAD

**Discussion by castasint** [microsoft/FHIR-Converter#239] [Ideas]
Hi - I see that the tool has been built as a command-line utility.  Is there any plan to possibly make it as an API like the Handlebars version? If not, is there any design reason/constraint as to why it was built as a CLI tool and not as a service/API? 

Also, is it advisable to take the tool and convert it into an API/service? Happy to answer any question. 

 I appreciate the effort that went into creating this awesome tool. 

Thanks,
Hanuma

**Comment by ranvijaykumar** [comment]
In addition to being available as a command line tool, the Liquid engine is also available in the [FHIR OSS server](https://github.com/microsoft/fhir-server), as well as the [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/) service in the form of $convert-data API. 

Please feel free to convert this tool into an API as per your needs.
